customizations/services.py

```diff
@@ -0,0 +1,192 @@
+from django.db import transaction, DatabaseError
+from django.shortcuts import (
+    get_object_or_404,
+)
+from customizations.utils import (
+    upload_file,
+    get_token,
+    create_webhook,
+    upload_file_dropbox,
+    get_unique_file_name,
+)
+from customizations.models import (
+    Customization,
+    TicketTemplate,
+    CustomEmail,
+    UserWebhook,
+    BaseTicketTemplate,
+)
+import re
+import cStringIO
+from PIL import Image
+from django.core.files.base import ContentFile
+
+
+def get_image(request_image_data, user):
+    img_io = cStringIO.StringIO()
+    image_base = request_image_data
+
+    image_data = re.sub(
+        '^data:image/.+;base64,', '', image_base
+    ).decode('base64')
+    image = Image.open(cStringIO.StringIO(image_data))
+
+    image.save(img_io, format=image.format, quality=100)
+    name_image_partner = get_unique_file_name(user, 'imagen.jpg')
+    img_content = ContentFile(
+        img_io.getvalue(),
+        name_image_partner
+    )
+    return img_content
+
+
+def get_request_fields(request):
+    fields = {}
+    fields['name_customization'] = request.POST.get('name')
+    fields['message'] = request.POST.get('message')
+    fields['select_design_template'] = request.POST.get(
+        'select_design_template'
+    )
+    fields['message_ticket'] = request.POST.get('message_ticket')
+    fields['show_event_sequence'] = request.POST.get(
+        'show_event_sequence'
+    ) == 'on'
+    fields['show_ticket_type_sequence'] = request.POST.get(
+        'show_ticket_type_sequence') == 'on'
+    fields['hide_ticket_type_price'] = request.POST.get(
+        'hide_ticket_type_price'
+    ) == 'on'
+    fields['footer_description'] = request.POST.get('footer_description')
+    fields['double_ticket'] = request.POST.get('double_ticket') == 'on'
+    fields['pdf_ticket_attach'] = request.POST.get('pdf_ticket_attach')
+    return fields
+
+
+def compose_customization(request):
+    if request.POST.get('image_data') is not None:
+        img_content = get_image(request.POST.get('image_data'), request.user)
+    else:
+        img_content = ''
+        img_content.name = ''
+    links = upload_file(request, 'logo')
+    fields = get_request_fields(request)
```

Comment:
Good approach here attempting to separate the app in layers, in the best case you could avoid have an HTTP request in this `domain` layer.